pez/src/main/java/globantWorkshop/services/implementation/BookService.java

```diff
@@ -30,31 +33,49 @@
 
     @Override
     public ArrayList<Book> getAllBooks() {
-        //Should Be implemented
         List<Book> books = new ArrayList<Book>();
+        try {
+            books = bookDao.getAllBooks();
+        } catch (Exception ex) {
```

Comment:
You should catch a more specific exception and also do something with it, like log the error not just throw it again